### PR TITLE
Updated copyright statement and stripped out README

### PR DESCRIPTION
**Pull request by dusktreader**
#### What
We needed to use the correct copyright statement with Scania. Also, stripped out the README so it just essentially points at the docs instead.

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Review by julianaklulo — APPROVED** [review]
I like the new README, nice and clean!

**Review by jamesbeedy — APPROVED** [review]
👍

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2020 OmniVector Solutions
+Copyright (c) 2022 Scania CV AB
+Copyright (c) 2022 Omnivector Solutions
```
Comment (heitorPB) [inline LICENSE]: Shouldn't the date be 2020-2022?

Comment (dusktreader) [reply LICENSE]: Updated in latest commit.

**Pull request merged**